gen/src/write.rs

```diff
@@ -236,13 +236,21 @@ fn write_struct<'a>(out: &mut OutFile<'a>, strct: &'a Struct, methods: &[&Extern
     writeln!(out, "#ifndef {}", guard);
     writeln!(out, "#define {}", guard);
     for line in strct.doc.to_string().lines() {
-        writeln!(out, "//{}", line);
+        if cfg!(feature = "doxygen") {
```

Comment:
Could you justify why not to always emit `///`? Is it because that would break anyone currently running Doxygen on cxx-generated headers in which the Rust comments are not "valid Doxygen" in some way?

Reply:
This PR is not a complete implementation for generating the Doxygen docs. But a small enhancement that is enough for fullfilling the minimal doxygen docs generate requirement. If we need more advance docs, maybe it is worth to add more features later on.